tests/predictionMarketContract.js

```diff
@@ -18,12 +18,13 @@ context('Prediction Market Contract', async () => {
   let marketId = 0;
   let outcomeIds = [0, 1];
   const ethAmount = 0.01;
+  let closeDateTime;
 
   context('Contract Deployment', async () => {
     it('should start the Application', mochaAsync(async () => {
       app = new Application({
-        web3Provider: 'https://kovan.infura.io/v3/5a3e570fd7e544779cf240da53d0d908',
-        web3PrivateKey: '0x2e0bf3d13116fbb5b323c8969f61cb9710da3d6df66ceff81a962e57b3701905'
+        web3Provider: process.env.WEB3_PROVIDER,
+        web3PrivateKey: process.env.WEB3_PRIVATE_KEY,
```

Comment:
@rsmarques This changes make sense?

Reply:
Yes! It was coded that way in the first place